connector/connect/bin/spark-connect

```diff
@@ -26,7 +26,14 @@ FWDIR="$(cd "`dirname "$0"`"/../../..; pwd)"
 cd "$FWDIR"
 export SPARK_HOME=$FWDIR
 
+# Determine the Scala version used in Spark
+SCALA_BINARY_VER=`grep "scala.binary.version" "${SPARK_HOME}/pom.xml" | head -n1 | awk -F '[<>]' '{print $3}'`
+SCALA_ARG=$(if [ "${SCALA_BINARY_VER}" == "2.13" ]; then echo "-Pscala-2.13"; else echo ""; fi)
```

Comment:
is there any side-effect if we simply use `-Pscala-${SCALA_BINARY_VER}` here?

Reply:
There is no functional difference. I did not use "-Pscala-2.12" as it was not explicitly used in https://spark.apache.org/docs/latest/building-spark.html So I followed the current example commands.

Reply:
IMO `-Pscala-${SCALA_BINARY_VER}` is quite unified and can reduce efforts when Spark decides to migrate scala 2.13 in default.

Reply:
Good point.